src/newcomponents/info/elements/ParticipantInChat.js

```diff
@@ -15,6 +31,19 @@ export default function ParticipantInChat({ userObject, isOwner }) {
           ? addPrefix(pathname + hash, "/profile")
           : addSuffix(pathname + hash, `/user?uid=${userObject._id}`)
       }
+      onContextMenu={(e) => {
+        e.preventDefault();
+        dispatch(
+          setList([
+            "infoUser",
+            isCurrentUser ? null : "newChat",
+            !isCurrentUserOwner || isCurrentUser ? null : "removeParticipant",
+          ])
+        );
+        dispatch(setCoords({ x: e.pageX, y: e.pageY }));
```

Comment:
can we have a single dispatch function to post this info at once?
Otherwise looks too complex